stable/grafana/values.yaml

```diff
@@ -430,14 +430,14 @@ dashboardImports:
   enabled: false
 
   ## How long should it take to commit failure
-  ## Default: 300
+  ## Default: 30
   ##
-  # activeDeadlineSeconds: 300
+  activeDeadlineSeconds: 30
 
   ## Docker image to run import process
-  ## Default: appropriate/curl:latest
+  ## Default: jgoclawski/wget:1.0
   ##
-  # image: appropriate/curl:latest
+  image: jgoclawski/wget:1.0
```

Comment:
Now I see why `wget` - promoting your own container 😄

Reply:
It's just a side-effect :) I couldn't find a simple image that would fit my needs.
Long story short: It's because of one of the points that I wanted to fix: "correctly fail job on errors and print output" mentioned in the PR description.

Long story below :)
With curl used without additional flags pods would always succeed, even if dashboard fetching or uploading would fail.
Example:
```
$ docker run --rm appropriate/curl curl https://grafana.com/api/dashboards/not-found
{
  "code": "NotFound",
  "message": "Dashboard not found"
}
$echo $?
0
```
The exit code is 0, which caused the job to think that the pod has succeeded. 
I've added "set -e" to the script to exit on first error, but it means that curl should exit with an error code in case of an error.

So I thought about adding `--fail` flag. It correctly sets the exit code but it doesn't print the body of the response, which might include important information. For example: when uploading dashboards Grafana may reject the dashboard if it's name is missing - we want to show that error, so that the user knows what went wrong.

```
$ docker run --rm appropriate/curl curl --fail https://grafana.com/api/dashboards/not-found
curl: (22) The requested URL returned error: 404 Not Found
$ echo $?
22
```

The exit code is OK, but the message is not there.

It turns out it's much cleaner to achieve with wget than with curl: https://stackoverflow.com/questions/22506024/get-page-output-with-curl-fail

So I switched to wget and tried a few most popular images... and they failed with TLS certificate validation.

```
$ docker run --rm inutano/wget wget -O- --content-on-error https://grafana.com/api/dashboards/not-found
...
ERROR: cannot verify grafana.com's certificate, issued by 'CN=COMODO RSA Domain Validation Secure Server CA,O=COMODO CA Limited,L=Salford,ST=Greater Manchester,C=GB':
  Unable to locally verify the issuer's authority.
To connect to grafana.com insecurely, use `--no-check-certificate'.
```

The same was for the other 5 most popular wget images on Docker Hub. So I've created my own, based on alpine:3.7 which works as intended.

```
$ docker run --rm jgoclawski/wget:1.0 wget -qO- --content-on-error https://grafana.com/api/dashboards/not-found
{
  "code": "NotFound",
  "message": "Dashboard not found"
}
$ echo $?
8
```